config/settings.yml

```diff
@@ -97,6 +97,7 @@
     :purge_window_size: 10000
 :ems:
   :ems_redhat:
+    :resolve_ip_addresses: true
```

Comment:
@adahms what is the process to add that text to the providers guide? Sending a pull request to some GitHub repository? Where? Will you take care of it or should I?